docs/commands.md

```diff
@@ -1,68 +1,103 @@
 # RedisTimeSeries Commands
 
-## TS.CREATE - create a new time-series
+## Create
+
+### TS.CREATE - create a new time-series
 ```sql
-TS.CREATE key [retentionSecs] [maxSamplesPerChunk] [labels]
+TS.CREATE key [RETENTION retentionSecs] [LABELS field value..]
 ```
 * key - key name for timeseries
 
 Optional args:
    * retentionSecs - max age for samples compared to current time (in seconds).
-      * Default: 0
+      * Default: the global retenionsecs configuration of the database. If not set, this is 0.
       * When set to 0, the series will not be trimmed at all
-   * maxSamplesPerChunk - how many samples to keep per memory chunk
-      * Default: 360
    * labels - set of key-value pairs that represent metadata labels of the key
 
-### Example
+#### Example
 ```sql
-TS.CREATE temperature 60 360 sensor_id=2 area_id=32
+TS.CREATE temperature RETENTION 60 LABELS sensor_id 2 area_id 32
 ```
 
-## TS.CREATERULE - create a compaction rule
+## Update
+
+### TS.ADD - append (or create and append) a new value to the series
 ```sql
-TS.CREATERULE sourceKey aggType bucketSizeSec destKey
+TS.ADD key timestamp value [RETENTION retentionSecs] [LABELS field value..]
 ```
-* sourceKey - key name for source time series
-* aggType - aggregation type one of the following: avg, sum, min, max, range, count, first, last
-* bucketSizeSec - time bucket for aggregated compaction,
-* destKey - key name for destination time series
-
-> DEST_KEY should be of a `timeseries` type, and should be created before TS.CREATERULE is called.
+* timestamp - unix timestamp (in seconds) or `*` for automatic timestamp (using the system clock)
```

Comment:
timestamp is a 64bit unsigned integer (so not just secs)

Reply:
@danni-m so milliseconds?